src/scripts/underground/Underground.ts

```diff
@@ -171,7 +171,7 @@ class Underground {
             default:
                 const type = item.valueType.charAt(0).toUpperCase() + item.valueType.slice(1); //Capitalizes string
                 const typeNum = PokemonType[type];
-                player._shardsCollected[typeNum](player._shardsCollected[typeNum]() + GameConstants.PLATE_VALUE);
+                App.game.shards.shardWallet[typeNum] += GameConstants.PLATE_VALUE;
```

Comment:
Watch out, here you increment the variable directly. This means that statistics aren't logged and possible bonuses aren't applied. Take a look at `App.game.shards.gainShards()` :)

Reply:
I missed that function somehow, change has been made!